docs/writing-tests/accessibility-testing.md

```diff
@@ -6,29 +6,37 @@ Accessibility is the practice of making websites inclusive to all. That means su
 
 Accessibility tests audit the rendered DOM against a set of heuristics based on [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) rules and other industry-accepted best practices. They act as the first line of QA to catch blatant accessibility violations.
 
-Storybook’s official [a11y addon](https://storybook.js.org/addons/@storybook/addon-a11y) runs accessibility audits while you’re developing components to give you a fast feedback loop. It's powered by Deque's [axe-core](https://github.com/dequelabs/axe-core), which automatically catches up to [57% of WCAG issues](https://www.deque.com/blog/automated-testing-study-identifies-57-percent-of-digital-accessibility-issues/).
-
 ![Accessibility testing](./accessibility-testing-storybook.gif)
 
-## Setup a11y addon
+## Accessibility testing with a11y addon
+
+Storybook provides an official [a11y addon](https://storybook.js.org/addons/@storybook/addon-a11y). Powered by Deque's [axe-core](https://github.com/dequelabs/axe-core), which automatically catches up to [57% of WCAG issues](https://www.deque.com/blog/automated-testing-study-identifies-57-percent-of-digital-accessibility-issues/).
+
+### Set up the a11y addon
+
+If you want to run accessibility tests using the [addon](https://storybook.js.org/addons/@storybook/addon-a11y/), you'll need additional steps to set it up. Documented below is our recommendation.
```

Comment:
> run accessibility tests

This phrase makes me think of automated testing, which is covered explicitly (and, importantly, _differently_) later. Maybe:

> check accessibility for your stories

Reply:
> Documented below is our recommendation.

It's not so much a recommendation as "the way to do it", right? The way this section shares a lot of phrasing with the 'Automate accessibility tests with test runner' section made me wonder if there was a copy error for a moment.

Reply:
So, we both of those comments in mind:

```suggestion
If you want to check accessibility for your stories using the [addon](https://storybook.js.org/addons/@storybook/addon-a11y/), you'll need to install it and add it to your Storybook.
```